ui/src/hooks/beta/useFilteredData.tsx

```
@@ -0,0 +1,46 @@
+        (filters.category === "All categories" ||
+          el.newCategory.toLowerCase() === filters.category ||
+          (filters.category === "whale + whale (ai)" &&
+            ["whale", "whale (ai)"].includes(el.newCategory.toLowerCase()))) &&
+        (filters.timeRange === allTime ||
+          filters.timeRange === customRange ||
+          Date.parse(el.timestampString) >= min) &&
+        (!filters.startDate ||
+          dayjs(el.timestampString).isSameOrAfter(filters.startDate, "day")) &&
+        dayjs(el.timestampString).isSameOrBefore(filters.endDate, "day") &&
+        (searchQuery === "" ||
+          (el.comments && el.comments.toLowerCase().includes(searchQuery)) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)